rep/db/db.py

```diff
@@ -44,15 +46,66 @@ def create_db(self):
                 unixTime INTEGER NOT NULL,
                 billNumber VARCHAR NOT NULL,
                 voteName VARCHAR NOT NULL,
-                repName VARCHAR NOT NULL,
+                repId INTEGER NOT NULL,
                 repVote VARCHAR NOT NULL,
                 rawVoteObjectId INTEGER NOT NULL,
                 FOREIGN KEY (rawVoteObjectId) REFERENCES raw_vote_object(id),
-                PRIMARY KEY (rawVoteObjectId, repName)
-            )
+                PRIMARY KEY (rawVoteObjectId, repId)
+            );
+
+        CREATE TABLE IF NOT EXISTS representative_info
+            (
+                id INTEGER PRIMARY KEY,
+                dist VARCHAR NULL,
+                officeCode VARCHAR NULL,
+                districtNumber VARCHAR NULL,
+                designatorCode VARCHAR NULL,
+                firstName VARCHAR NULL,
+                middleInitial VARCHAR NULL,
+                lastName VARCHAR NULL,
+                suffix VARCHAR NULL,
+                commonlyUsedName VARCHAR NULL,
+                homeStreetAddress VARCHAR NULL,
+                homeCity VARCHAR NULL,
+                homeState VARCHAR NULL,
+                homeZipCode VARCHAR NULL,
+                homePhone VARCHAR NULL,
+                capitolStreetAddress VARCHAR NULL,
+                capitolCity VARCHAR NULL,
+                capitolPhone VARCHAR NULL,
+                room VARCHAR NULL,
+                roomNumber VARCHAR NULL,
+                committeesChaired VARCHAR NULL,
+                committeesViceChaired VARCHAR NULL,
+                rankingMember VARCHAR NULL,
+                committeeMember1 VARCHAR NULL,
+                senatorRepresentative VARCHAR NULL,
+                party VARCHAR NULL,
+                title VARCHAR NULL,
+                gender VARCHAR NULL,
+                businessPhone VARCHAR NULL,
+                email VARCHAR NULL,
+                fax VARCHAR NULL,
+                prison VARCHAR NULL,
+                url VARCHAR NULL,
+                committeeCodes VARCHAR NULL
+            );
         ''')
 
         conn.commit()
+
+        filename = 'database/data/LegislatorDatabase_2020.csv'
+        with open(filename) as csv_file:
+            reader = csv.reader(csv_file)
+            next(reader)
+            for field in reader:
+                representative_info_dao = RepresentativeInfoDao()
+                # I can't get the auto-incrementing ID to work unless I give
+                # a -1 as the first list entry for the dataclass.
+                # I know this is bad!! But I don't know how else to fix it
+                field.insert(0, -1)
+                representative_info_dao.write(representative_info_dao._map_result(field))
```

Comment:
won't this insert every single row from the csv each time it runs? (which will be every time the service starts). If that's the case, you'll either need to check if the record exists before you write it, or you can add a `UNIQUE` constraint to the table and use `INSERT IGNORE INTO` or something instead